progress.go

```diff
@@ -111,40 +126,23 @@ func (p *Progress) reporter() {
 	}
 }
 
-// Reset resets all statistic counters to zero.
-func (p *Progress) Reset() {
-	if p == nil {
-		return
-	}
-
-	if !p.running {
-		panic("resetting a non-running Progress")
-	}
-
-	p.curM.Lock()
-	p.cur = Stat{}
-	p.curM.Unlock()
-}
-
 // Done closes the progress report.
 func (p *Progress) Done() {
 	if p == nil || !p.running {
 		return
 	}
 
-	if p.running {
```

Comment:
this check was unnecessary since you already check for `!p.running` above and then `return`